connector/src/main/java/org/jboss/as/connector/subsystems/datasources/GetInstalledDriverOperationHandler.java

```diff
@@ -59,18 +57,13 @@ public class GetInstalledDriverOperationHandler implements OperationStepHandler
 
     public static final GetInstalledDriverOperationHandler INSTANCE = new GetInstalledDriverOperationHandler();
 
-    private final ParametersValidator validator = new ParametersValidator();
-
     private GetInstalledDriverOperationHandler() {
-        validator.registerValidator(DRIVER_NAME.getName(), new StringLengthValidator(1));
     }
 
     @Override
     public void execute(final OperationContext context, final ModelNode operation) throws OperationFailedException {
 
-        validator.validate(operation);
-
-        final String name = operation.require(DRIVER_NAME.getName()).asString();
+        final String name = operation.require(INSTALLED_DRIVER_NAME.getName()).asString();
```

Comment:
Using ModelNode.require is an anti-pattern, although it did no harm here as the validator already confirmed the parameter.

ModelNode.get should be used.